src/components/ResourceCard/ResourceCard.styles.js

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import mediaQuery from '../../utils/mediaQuery';
+
+export const Banner = styled.div`
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  opacity: 0.6;
+`;
+
+export const Content = styled(Link)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  background: var(--white);
+  border: ${({ color }) => `0.1rem solid var(--${color})`};
+  border-radius: var(--smallest-space);
+
+  overflow: hidden;
+  position: relative;
+  width: 32.8rem;
```

Comment:
It is not working properly in small mobile mode, do we have to change it to max-width?

Reply:
I refactored the code. I set `max-width: 60rem;` for the container and removed some redundant code from wrapper holding Banner so now it is only for` opacity` purpose.